__tests__/contact/contact.query.test.js

```diff
@@ -84,29 +83,27 @@ describe('Contacts queries', () => {
         `,
         variables: {
           skip: 1,
-          limit: 6,
+          limit: 1,
         },
       })
       .catch(e => e);
 
     expect(res.data.getContacts).toBeDefined();
-    expect(res.data.getContacts).toContainEqual({
+    expect(res.data.getContacts.items).toContainEqual({
       __typename: 'Contact',
       address: [
-        { __typename: 'Language', lang: 'uk', value: 'Вулиця' },
-        { __typename: 'Language', lang: 'en', value: 'Street' },
+        { __typename: 'Language', lang: 'uk', value: 'Вулиця 4' },
```

Comment:
Please, use here values from the `contact` instead of plain hardcode literals. It would be much easier and safe to changes some values in `contact` and be sure that it is changed in tests too. Use hardcode values only for properties that you cannot take from `contact`